overview.md

```diff
@@ -0,0 +1,180 @@
+# STAC Overview
+
+There are three component specifications that together make up the core SpatioTemporal Asset Catalog specification.
+Each can be used alone, but they work best in concert with one another. The [STAC API specification](https://github.com/radiantearth/stac-api-spec) 
+builds on top of that core, but is out of scope for this overview. Items are the core atomic unit, representing a 
+single spatiotemporal asset as GeoJSON so it can be searched. Catalogs are structural elements, used to group Items
+and Collections. Collections *are* catalogs, that add more required metadata and describe a group of related Items.
+For more on the differences see the [section below](#catalogs-vs-collections)
+
+## Item Overview
+
+Fundamental to any SpatioTemporal Asset Catalog, an [Item](item-spec/item-spec.md) represents an atomic collection of inseparable 
+data and metadata. A STAC Item is a [GeoJSON](http://geojson.org/) [Feature](https://tools.ietf.org/html/rfc7946#section-3.2)
+and can be easily read by any modern GIS or geospatial library, and it describes a [SpatioTemporal Asset](#what-is-a-spatiotemporal-asset). 
+The STAC [Item JSON specification](item-spec/item-spec.md) includes additional fields for:
+
+* the time the asset represents;
+* a thumbnail for quick browsing;
```

Comment:
There's no field for thumbnails, that's part of the assets?!

Reply:
I was describing in a more general sense. I suppose I could just say 'The STAC Item JSON specification includes additional information (constructs?) for:' 

Like I don't think at this point we need to get technical about exactly where these things are. But it's a fair point that fields could imply that it's an actual top level field.

Reply:
Yes, I think replacing field with something else would help. :-)